doc/langref.html.in

```diff
@@ -7015,7 +7015,9 @@ test "try to compare bools" {
       </p>
       {#code_begin|syntax|compiler_generated_function#}
 fn max(a: bool, b: bool) bool {
-    return a or b;
+    {
```

Comment:
This isn't valid zig.

I think it would be more clear to write

```zig
fn max(a: bool, b: bool) bool {
  return blk: {
    break :blk a or b;
  }
}
```